graphcore_cloud_tools/paperspace_utils/symlink_datasets_and_caches.py

```diff
@@ -431,6 +479,9 @@ def symlink_arguments(parser=argparse.ArgumentParser()) -> argparse.ArgumentPars
         default=str(Path(".").resolve().parent / "settings.yaml"),
         help="Path to gradient settings.yaml file",
     )
+    parser.add_argument(
+        "--max-retries", default=1, type=int, help="Maximum number of s3 download retries in case of failures"
```

Comment:
Can you call it `--max-attempts` and set the number to two? I find the notion of maximum retries a bit unclear